chsdi/templates/htmlpopup/base.mako

```diff
@@ -12,6 +12,7 @@
   isExtended = c.get('isExtended')
   isIframe = c.get('isIframe')
   isGridLayer = c.get('isGridLayer')
+  c['htmlpopup_class'] = c['layerBodId'].replace('.', '')
```

Comment:
can't really happen it's done much earlier https://github.com/geoadmin/mf-chsdi3/blob/master/chsdi/lib/validation/features.py#L50